modules/auxiliary/gather/ldap_query.rb

```diff
@@ -206,6 +206,20 @@ def output_data_csv(entries)
     generate_rex_tables(entries, 'csv')
   end
 
+  def normalize_entries(entries)
+    cleaned_entries = []
+    entries.each do |entry|
+      # Convert to a hash so we get only the data we need.
+      entry = entry.to_h
+      entry_keys = entry.keys
+      entry_keys.each do |key|
+        entry[key] = entry[key].map { |v| Rex::Text.to_hex_ascii(v) }
```

Comment:
Looks like we'll want to remove this for genuine UTF-8 characters, for the account `wîth.dìáçriticš`


Metasploit:
```
msf6 auxiliary(gather/ldap_query) > run rhost=192.168.123.13 username=Administrator@adf3.local password=p4$$w0rd

... etc ...

CN=w\xc3\xaeth.d\xc3\xac\xc3\xa1\xc3\xa7ritic\xc5\xa1 CN=Users DC=adf3 DC=local
===============================================================================

 Name                Attributes
 ----                ----------
 displayname         w\xc3\xaeth.d\xc3\xac\xc3\xa1\xc3\xa7ritic\xc5\xa1
 name                w\xc3\xaeth.d\xc3\xac\xc3\xa1\xc3\xa7ritic\xc5\xa1
 samaccountname      w\xc3\xaeth.d\xc3\xac\xc3\xa1\xc3\xa7ritic\xc5\xa1
 useraccountcontrol  66048
 userprincipalname   w\xc3\xaeth.d\xc3\xac\xc3\xa1\xc3\xa7ritic\xc5\xa1@adf3.local
```

Expected:

```
displayname       wîth.dìáçriticš
... etc...
userprincipalname wîth.dìáçriticš@adf3.local
```

Similar to other tools:

<img width="782" alt="image" src="https://user-images.githubusercontent.com/60357436/188766593-fd72b94d-ed75-4bae-b5f0-8d300701fd16.png">

Although it looks like ldapsearch doesn't handle this scenario intuitively either; The internet suggests to pipe to perl to have it handle the characters correctly for its base64 decoding https://access.redhat.com/solutions/43621 🤷 

```
# w\C3\AEth.d\C3\AC\C3\A1\C3\A7ritic\C5\A1, Users, adf3.local
dn:: Q049d8OudGguZMOsw6HDp3JpdGljxaEsQ049VXNlcnMsREM9YWRmMyxEQz1sb2NhbA==
objectClass: top
objectClass: person
objectClass: organizationalPerson
objectClass: user
```